android/src/main/java/cl/json/social/ShareIntent.java

```diff
@@ -124,6 +127,68 @@ public void open(ReadableMap options) throws ActivityNotFoundException {
             }
         }
 
+        if (socialType.equals("instagramstories")) {
+            if (ShareIntent.hasValidKey("method", options)) {
+                String method = options.getString("method");
+                switch (method) {
+                    case "shareBackgroundImage": {
+                        if (ShareIntent.hasValidKey("backgroundImage", options)) {
+                            this.backgroundAsset = new ShareFile(options.getString("backgroundImage"), "image/jpeg", "background", this.reactContext);
+                            this.getIntent().setDataAndType(backgroundAsset.getURI(), backgroundAsset.getType());
+                            this.getIntent().addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
+
+                            if (ShareIntent.hasValidKey("attributionURL", options)) {
+                                this.getIntent().putExtra("content_url", options.getString("attributionURL"));
+                            }
+                        } else {
+                            throw new java.lang.IllegalArgumentException("backgroundImage is empty");
```

Comment:
We could send an error message instead. By doing this, the app will crash, right?

Reply:
No. This exception will be cached in RNShareModule.java.
Error message will be send from there